content/ember/v1/access-to-instances.md

```diff
@@ -0,0 +1,52 @@
+---
+id: deprecate-access-to-instances-in-initializers
+title:  Access to Instances in Initializers
+until: ''
+since: '1.11'
+---
+
+Previously, initializers had access to an object that allowed them to
+both register new classes and get instances of those classes.
+
+If you have an initializer that gets instances of a class, you need to
+change it to use an instance initializer.
+
+Change code that looks like this:
+
+```js
```

Comment:
Needs to be updated to new syntax for code highlighting, if you would like I can contribute my regexing 💪 😂

Reply:
that would be amazing ;) I keep forgetting we need to add this new syntax :) definitly one to add to the list of readme for new contributors :)